HashSet.java

```diff
@@ -0,0 +1,75 @@
+// TC=O(1)
+// SC=
+// as the input is 1000000 divided the array into blocks of 1000 and each
+// block can be assigned an array of 1000 buckets if necessary
+// for performing any operation know the index of bucket, while inserting in the bucket
+// find the index of where boolean value needs to be stored, if there is a collision create
+// new array and insert the key to the index obtained by second hashing
+
+// Issue:
+// class, interface, or enum expected
+//
+//  not a statement: if(boolean[b1][b2]==true){
```

Comment:
returning of a boolean is preferred done without equality operators. You can text me on Slack for further help/clarification if needed!